packages/kbn-safer-lodash-set/BUILD.bazel

```diff
@@ -1,9 +1,6 @@
 load("@build_bazel_rules_nodejs//:index.bzl", "js_library", "pkg_npm")
```

Comment:
The other `BUILD.bazel` files has been deleted, but this one hasn't. How come?

Reply:
See the last questing in the description, TLDR there are still a few webpack bundles like `@kbn/ui-shared-deps-src` which include some packages, those packages need a BUILD.bazel file for now, but new packages should not be added to these bundles and we're going to start working to remove them next.